makefile

```diff
@@ -1,6 +1,6 @@
 # Common prefixes for installation directories.
 # See: https://www.gnu.org/software/make/manual/make.html
-prefix = /usr/local
+prefix = /usr
```

Comment:
If you look at `16.5 Variables for Installation Directories` under the link above you see that the default should be `/usr/local/`. Since the GNU defaults are so widespread it's kind of what one expects as well, meaning that any deviation ends up surprising to end users.

Would your issue be solved if #72 was merged?